<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_update</class>
 <widget class="QDialog" name="fct_update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update BD V1</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_where">
    <item>
     <widget class="QLabel" name="label_where">
      <property name="text">
       <string>WHERE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_where"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_indication">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Indication : Completer les lignes correspondante pour former un UPDATE valide</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_set">
    <item>
     <widget class="QLabel" name="label_set">
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_set"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_exemple">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exemple : UPDATE LesSportifs_base SET nomSp = 'Dupont', prenomSp = 'Jean WHERE nomSp='dupont'</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_update_info">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>782</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_condition">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_condition">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>551</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Si l'update comprend une condition &quot;WHERE&quot; alors la case doit être cochée</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_update_query">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>782</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_update">
    <item>
     <widget class="QLabel" name="label_update">
      <property name="text">
       <string>UPDATE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_update"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_update</sender>
   <signal>clicked()</signal>
   <receiver>fct_update</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
